tweak: add patch boxes and thick tape roll to the taipan

Mahimka42 opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
добавляет 3 толстой изоленты в пыточную тайпана и 5 патчниц в медицинский отдел тайпана
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
https://discord.com/channels/617003227182792704/1279745641156972599/1281069257827946588
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений
![image](https://github.com/user-attachments/assets/8d0b8183-edad-460a-993e-967b9c778ea7)
![image](https://github.com/user-attachments/assets/b32dac34-b053-4ce9-86f5-7f83a304c8c4)

<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

## Тесты
Зашел на тайпан, все проверил, все работает.
<!-- Как вы тестировали свой код. Ревьеюру будет проще, если он будет знать какие действия вы совершали, проверяя свой ПР. -->

Mahimka42 committed:
Update syndicatebase.dmm

Daeberdir commented on _maps/map_files/generic/syndicatebase.dmm:
```diff
@@ -10637,6 +10656,10 @@
 	icon_state = "dark"
 	},
 /area/syndicate/unpowered/syndicate_space_base/engineering/storage)
+"jHV" = (
+/obj/item/stack/tape_roll/thick,
+/turf/space,
+/area/space)
```
Явный миссклик.

Daeberdir commented on _maps/map_files/generic/syndicatebase.dmm:
```diff
@@ -13637,6 +13660,9 @@
 /obj/machinery/light/small{
 	dir = 8
 	},
+/obj/item/stack/tape_roll/thick,
+/obj/item/stack/tape_roll/thick,
+/obj/item/stack/tape_roll/thick,
```
Тебе проще тут, вместо 3 мотков, сделать 1, но с тройным "количеством" использований.

Daeberdir commented on _maps/map_files/generic/syndicatebase.dmm:
```diff
@@ -6415,6 +6415,25 @@
 	},
 /obj/item/pen,
 /obj/item/hand_labeler,
+/obj/item/storage/pill_bottle/patch_pack{
+	pixel_y = 16
+	},
+/obj/item/storage/pill_bottle/patch_pack{
+	pixel_y = 16;
+	pixel_x = -2
+	},
+/obj/item/storage/pill_bottle/patch_pack{
+	pixel_y = 16;
+	pixel_x = -4
+	},
+/obj/item/storage/pill_bottle/patch_pack{
+	pixel_y = 16;
+	pixel_x = -6
+	},
+/obj/item/storage/pill_bottle/patch_pack{
+	pixel_y = 16;
+	pixel_x = -8
+	},
```
Это всё дело замени тупо коробкой с этими пачечками.

Mahimka42 replied:
я тоже думал коробку закинуть, но по предложе было 5

Daeberdir replied:
Значит сделай коробку с пятью. В любом случае, полная коробка никому бы там не навредила, ибо это Тайпан.

Daeberdir reviewed (commented):
Залей спрайт скотча без числа, чтобы его было видно на мапе, а то мапдифф слепой.

Mahimka42 committed:
Update syndicatebase.dmm

Mahimka42 committed:
Update syndicatebase.dmm

Mahimka42 committed:
Update syndicatebase.dmm

Mahimka42 commented:
блять, че я сделал

Daeberdir committed:
Brigadier's tweaks.

- Убрал миссклики зоны космоса.
- Крепкий скотч спавнится в тройном размере, а не три штуки.
- Фиксы NODATA спрайтов на мапе Тайпана.
- Пораскиданы обьекты на столах, которые были задеты автором.

Daeberdir reviewed (approved):
Внёс правки, включая небольшой фикс иконок для сдмм. Скотчу стоило и максимальное количество прописать, ибо остаток количества оно всё равно спавнило бы, как отдельные мотки. Для фокусирования окна на мапе в СДММе используй СКМ, а не ЛКМ, ибо довольно много мисскликов, которые создадут баги. Обьекты при настакивании в одно место лучше либо раздвигать в стороны, либо перемещать их в списке, чтобы мелкие спавнились визуально выше больших.
